release/models/bgp/openconfig-bgp-policy.yang

```diff
@@ -229,6 +236,19 @@ module openconfig-bgp-policy {
       the IGP cost using the enum (predefined value).";
   }
 
+  typedef bgp-as-mode-type {
```

Comment:
Although only used in policy, I do think this should be moved to release/models/bgp/openconfig-bgp-types.yang

Reply:
Moved `bgp-as-builtin-type ` to `openconfig-bgp-types.yang`. Thanks for the suggestion.